cmd/podman/common/create_opts.go

```diff
@@ -377,7 +387,7 @@ func ContainerCreateToContainerCLIOpts(cc handlers.CreateContainerConfig) (*Cont
 		cliOpts.MemoryReservation = strconv.Itoa(int(cc.HostConfig.MemoryReservation))
 	}
 
-	if cc.HostConfig.MemorySwap > 0 {
+	if cc.HostConfig.MemorySwap > 0 && !rootless.IsRootless() {
```

Comment:
These are valid for rootless, if the cgroupv2 is configured correctly.  @giuseppe WDYT?

Reply:
ok, lets change the conditional then.